test/WebSites/RoutingWebSite/Controllers/TeamController.cs

```diff
@@ -63,8 +63,8 @@ public ActionResult GetAllTeams(int notRelevant)
             return Content(Url.Action(), "text/plain");
         }
 
-        [HttpGet("/TeamName/{*Name}/")]
-        public ActionResult GetTeam(string name = "DefaultName")
```

Comment:
This is inconsistent with how conventional routes work with catchalls and inconsistent with our model binding works.